src/WooCommerce.php

```diff
@@ -1412,4 +1460,36 @@ public static function disableRelatedProducts(): void {
     }
   }
 
+  /**
+   * Retrives the repricing options dropdown values
+   * 
+   * @param WC_Product $product
+   *   The product for which values must be returend
+   * 
+   * @return array
+   *   Dropdown values
+   */
+  public static function getRepricingOptionsDropdownValues(\WC_Product $product) {
```

Comment:
@alinistrate — in my opinion this is just an internal helper method created only to avoid code duplication, since it would never get called from another third party instance, we might just as well declare it as `private`.
```suggestion
  private static function getRepricingOptionsDropdownValues(\WC_Product $product) {
```